### PR TITLE
docs: add `literalinclude` for curlrequest -> debug

### PR DESCRIPTION
**Pull request by totoprayogo1916**
before:
image

after:
image

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/libraries/curlrequest/034.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+$response->request('GET', 'http://example.com', ['debug' => true]);
```
Comment (kenjis) [inline user_guide_src/source/libraries/curlrequest/034.php:3]: This is not your mistake, but isn't it `$client->request(`?

Comment (kenjis) [reply user_guide_src/source/libraries/curlrequest/034.php:3]: Also all the other sample code in [Request Options](https://codeigniter4.github.io/CodeIgniter4/libraries/curlrequest.html#request-options).

Comment (totoprayogo1916) [reply user_guide_src/source/libraries/curlrequest/034.php]: updated

**Review by kenjis — APPROVED** [review]
Thank you!

**Pull request merged**